subsys/bluetooth/controller/Kconfig.ll_sw_split

```diff
@@ -1052,6 +1086,10 @@ config BT_MAYFLY_YIELD_AFTER_CALL
 	  If set to 'n', all pending mayflies for callee are executed before
 	  yielding
 
+config BT_TICKER_CNTR_FREE_RUNNING
+	# Hidden options to use free running counter in ticker implementation
+	bool
```

Comment:
Did you consider the approach of using `def_bool` instead of `select`?

Reply:
no, not aware of it, but will give a try in the future iterative PR with bsim port/coverage is functional.

Reply:
It's basically just `default`ing the value of `BT_TICKER_CNTR_FREE_RUNNING` rather than `select`ing it. 

It's not necessarily better, but worth a consideration